packages/@okta/vuepress-site/docs/reference/api/rulesets/index.md

```diff
@@ -0,0 +1,979 @@
+---
+title: RuleSets
+category: management
+---
+
+# Rule Sets
+
+<ApiLifecycle access="ea" />
+
+Rule Sets are ordered sets of rules that support more granular targeting and unioning so that matched rules can either contribute to the final result or possibly deny the action.
+
+They allow you to define requirements for multiple entities in a more reusable fashion. For example, a particular App may have a certain set of factor enrollment requirements, while a UserType may have a different set. It should be easy to take the union of the rules governing each.
+
+Rule Sets are exposed under the familiar [Okta Policy API](/docs/reference/api/policy).
+
+We distinguish between Policies and RuleSets based on the `type` field in the Policy API. Rule Set types always begin with the prefix `Okta:` (such as `Okta:SignOn`).
+
+Rule Sets also introduce the concept of `Mappings`, which represent the governance relationship of a policy over a given resource.
+
+Currently Okta allows a single Resource Mapping for each Rule Set, but the API will naturally support multiple mappings in the future.
```

Comment:
Are we 100% comfortable making this commitment in writing?

Reply:
I would remove after the `,`